packages/contracts-bedrock/package.json

```diff
@@ -77,7 +77,7 @@
     "dotenv": "^16.0.0",
     "ds-test": "https://github.com/dapphub/ds-test.git#9310e879db8ba3ea6d5c6489a579118fd264a3f5",
     "ethereum-waffle": "^3.0.0",
-    "forge-std": "https://github.com/foundry-rs/forge-std.git#5bafa16b4a6aa67c503d96294be846a22a6f6efb",
```

Comment:
The `yarn.lock` wasn't updated?

Reply:
It doesn't look like `contracts-bedrock` has a lockfile.

Reply:
I though the global `yarn.lock` should be updated

Reply:
Ah gotcha, wasn't updating automatically- manually bumped it 👍